graphql/schemabuilder/pagination.go

```diff
@@ -149,6 +149,18 @@ type PaginationInfo struct {
 	Pages          []string
 }
 
+// PostProcessOptions is used to instruct Thunder to perform additional operations on the output
+// in the case of manually paginated resolvers.
+// This struct should be returned as the second result from a manually-paginated field func.
+type PostProcessOptions struct {
+	// Whether or not Thunder should apply filtering on the output.
+	ApplyTextFilter bool
```

Comment:
What happens if thunder applies the text filter and filters out too many items?

Reply:
Resolvers should set this option with the possibility in mind that an arbitrary number of items will get filtered out.

A use case is when externally-managed resolvers determine they would like to fall back  to Thunder's filtering and fetch the entire data range (disregarding the `First` parameter). Then, Thunder's potential filtering of data such that the resulting set's length is less than the `First` parameter is just an inherent characteristic of the query and data store - there are just not enough items that fit the provided filter.

Reply:
Ah, that makes a lot more sense!

Reply:
Can we maybe add a comment explaining when to use it

Reply:
👍